ui/components/app/terms-of-use-popup/index.scss

```diff
@@ -0,0 +1,35 @@
+.popover-wrap.terms-of-use__popover {
+  .terms-of-use {
+    &__content {
+      display: flex;
+      flex-direction: column;
+    }
```

Comment:
Since, content is a box internally, we should use Box utility props to add the CSS.
```suggestion

```